### ISSUE TITLE
chore: revert message push log back to trace

### ISSUE THREAD

**Issue by richard-ramos** [waku-org/nwaku#1974]
In https://github.com/waku-org/nwaku/pull/1965 for debugging message pushes during dogfooding, logs where modified/added with debug level. Once dogfooding is complete, the log level for these messages must be changed to `trace` to reduce noise on logs

**Comment by gabrielmer** [comment]
Well we now are turning that log into `notice` in https://github.com/waku-org/nwaku/pull/2737 lol

This is almost a year old. I would honestly close this issue and if we see it's too noisy and has to be changed, open a new issue then.
WDYT @Ivansete-status ?

**Comment by Ivansete-status** [comment]
I agree with your comment @gabrielmer
